copulas/multivariate/tree.py

```diff
@@ -418,9 +446,55 @@ def get_likelihood(self, uni_matrix):
             right_ing = list(self.D - self.parents[1].D)[0]
             left_u = uni_matrix[self.L, left_ing]
             right_u = uni_matrix[self.R, right_ing]
-        cop = Bivariate(self.name)
-        cop.set_params(theta=self.theta)
+        cop = Bivariate.from_dict(copula_type=self.name, theta=self.theta)
```

Comment:
Please add an empty line above this one

Reply:
Also, here, instead of calling from_dict, create the instance of the copula first and then set the theta value directly as an attribute:

```
cop = Bivariate(self.name)
cop.theta = self.theta
```